### PR TITLE
Storyboard refuge_zh-hans.md file added

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione,

Adding storyboard refuge_zh-hans.md file. Sign on behalf of @XMRNoblesse. This should be ready for review. 

>:)

### CODE REVIEW

**Comment by lh1008** [comment]
Pinging @xjmzx @Lafudoci @almutsim. If anyone can make the review would be of great help. Thank you.

>:)

**Comment by lh1008** [comment]
@XMRNoblesse there were some comments left. Could you please check them out?
Maybe @xjmzx or @Lafudoci can help us out?

File: monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# 门罗币：于严苛的民事没收法之下保护你（第一部分）
+*2019年4月15日*  
+Obiter Cryptum  
+
+
+政府和第三方机构令人发指、系统性的权力滥用每天都在发生，虽然你可能没有亲身经历过，但它们始终存在并且异常活跃。虽然我们作为人的基本人权根据《世界人权宣言》(http://www.un.org/en/universal-declaration-human-rights/)和各种宪法得以确立和保障，但是这些权利总是或明或暗地被侵犯和无视。
```
Comment (Lafudoci) [inline monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md]: To make it in valid markdown format, it should be:
`虽然我们作为人的基本人权根据[《世界人权宣言》](http://www.un.org/en/universal-declaration-human-rights/)和各种宪法得以确立和保障。`

File: monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# 门罗币：于严苛的民事没收法之下保护你（第一部分）
+*2019年4月15日*  
+Obiter Cryptum  
+
+
+政府和第三方机构令人发指、系统性的权力滥用每天都在发生，虽然你可能没有亲身经历过，但它们始终存在并且异常活跃。虽然我们作为人的基本人权根据《世界人权宣言》(http://www.un.org/en/universal-declaration-human-rights/)和各种宪法得以确立和保障，但是这些权利总是或明或暗地被侵犯和无视。
+
+言论自由、选举权和自治权并不是仅仅在那些[三十亿多的穷苦人民]中较难实现(http://www.globalissues.org/article/26/poverty-facts-and-stats) （他们每天的花销仅在2.5美元内），对于中产阶级和富人阶层也是如此。他们可能会被政府、社区的集体行动或各种各样的审查制度所限制。
```
Comment (Lafudoci) [inline monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md]: To make it in valid markdown format, it should be:
`选举权和自治权并不是仅仅在那些[三十亿多的穷苦人民](http://www.globalissues.org/article/26/poverty-facts-and-stats)中较难实现（他们每天的花销仅在2.5美元内）`

File: monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md]: Added. thanks.

File: monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/zh-hans/storyboard/refuge_zh-hans.md]: Added also.

**Comment by lh1008** [comment]
Added suggested changes @Lafudoci. Just needs approving if there's nothing else to change to be merged. 

>:)

**Comment by lh1008** [comment]
Oh, @al-mutasim the suggested double check here:

>1. In Case 2 of the Civil Forfeiture Cases, the woman and her son were on a house-hunting trip in Florida, looking for a new home (rather than hunting game). Please double check that this is clear.

And here:

> 2. Under "How Monero Can Serve the People & Insulate From Inequitable Civil Forfeiture Laws" the civil forfeiture laws are not related to Monero, but rather Monero counters them. A quick check to be sure this is clear would be helpful.

Can anyone translate those? We could add them to the existing file so we can merge this PR. 

@Lafudoci @XMRNoblesse @xjmzx

**Comment by Lafudoci** [comment]
>  1. In Case 2 of the Civil Forfeiture Cases, the woman and her son were on a house-hunting trip in Florida, looking for a new home (rather than hunting game). Please double check that this is clear.

al-mutasim is correct, the translation here means a animal hunting trip. 

so `尽管Gentile解释到这是为了去佛罗里达州的一次狩猎旅行而准备的` could be:

`尽管Gentile解释到这是为了去佛罗里达州找新房而准备的`

>  1. Under "How Monero Can Serve the People & Insulate From Inequitable Civil Forfeiture Laws" the civil forfeiture laws are not related to Monero, but rather Monero counters them. A quick check to be sure this is clear would be helpful.

This part looks good to me.

**Comment by Lafudoci** [comment]
@lh1008 I reviewed the rest content. It looks good to me.
But please note I'm not native speaker, so I can only tell they are understandable, but can't guarantee the wording is native.

**Comment by lh1008** [comment]
Oh, this was merged without the changes. I will revert so I can apply @Lafudoci's changes.

**Comment by erciccione** [comment]
My mistake, i thought this was ready.

**Comment by lh1008** [comment]
No worries >:)

**Pull request merged**